src/libs/actions/Card.ts

```diff
@@ -381,6 +381,24 @@ function startIssueNewCardFlow(policyID: string) {
     API.read(READ_COMMANDS.START_ISSUE_NEW_CARD_FLOW, parameters);
 }
 
+function issueExpensifyCard(policyID: string, feedCountry: string, {assigneeEmail, limit, limitType, cardTitle, cardType}: IssueNewCardData) {
+    const parameters = {
+        policyID,
+        assigneeEmail,
+        limit,
+        limitType,
+        cardTitle,
+    };
+
+    if (cardType === CONST.EXPENSIFY_CARD.CARD_TYPE.PHYSICAL) {
+        API.write(WRITE_COMMANDS.CREATE_EXPENSIFY_CARD, {...parameters, feedCountry});
+        return;
+    }
+
+    // eslint-disable-next-line rulesdir/no-multiple-api-calls
+    API.write(WRITE_COMMANDS.CREATE_ADMIN_ISSUED_VIRTUAL_CARD, parameters);
```

Comment:
We have to include `domainName` (`expensify_policy<policyID>.exfy`) for the virtual card.

The API is shared with backend and it [does not accept the policyID](https://github.com/Expensify/Web-Expensify/blob/28daaa9937008a184443787e81a7482ae9547eb6/lib/Auth.php#L9998-L10007) so we have to identify the domain using the policyID @MariaHCD

Reply:
@mountiny so I should replace policyID with domainName here?

Reply:
I will be updating the CreateAdminIssuedVirtualCard command to accept policyID so I think we can leave this as it is.